README.md

```diff
@@ -22,6 +22,7 @@ Elements are written to make it easy to work in a variety of web frameworks. If
 | [mux-player-react](packages/mux-player-react)     | A Mux-flavored React media player component   | React         | 1.x.x 🎉                |
 | [mux-uploader](packages/mux-uploader)             | A Mux-flavored HTML5 media uploader component | Web Component | Public Beta             |
 | [mux-uploader-react](packages/mux-uploader-react) | A Mux-flavored React media uploader component | React         | Public Beta             |
+| [mux-viewer-count](packages/mux-viewer-count)     | A Mux-flavored HTML5 viewer count component   | Web Component | Public Alpha            |
```

Comment:
I know this is gonna feel like dropping 💣 right here when this is all done but I'm wondering if `<mux-viewer-count>` might make people think it's the _total_ views count (which people have asked for).

Should we rename to:

1. `<mux-live-viewer-count>`?
2. `<mux-active-viewer-count>`?

I think num 2 

Easier to rename it now before it's published, so worth getting it right

Reply:
I think #2.  Making that change now.